lib/appsignal/config.ex

```diff
@@ -111,7 +110,7 @@ defmodule Appsignal.Config do
     |> do_active?
   end
 
-  defp do_active?(%{valid: true, active: true}), do: true
```

Comment:
This could be a problem. Users can set `active: true`, but we should also check that the config is considered valid by the config module.

Maybe we can store the "valid" state somewhere else and fetch it in this function later?

Reply:
I created a helper function so we know if the config is valid